api/survey/routes.js

```diff
@@ -158,10 +158,80 @@ router.get("/results", passport.isOrganization, function(req, res) {
   if (!req.user.organization.current_survey) {
     return res.status(400).send("Organization does not have a current survey")
   }
+  console.log("SURVEY = " + req.user.organization.current_survey);
   Response.find({survey: req.user.organization.current_survey}, function(err, results) {
     if (err) return res.status(400).send(err);
     return res.json(results);
   });
 });
 
+router.get("/all-results", passport.isOrganization, function(req, res) {
+  if (!req.user.organization.current_survey) {
+    return res.status(400).send("Organization does not have any past surveys")
+  }
+  //get current org
+  var currentOrg = req.user.organization._id
+  var allResponses = [];
+  var surveysCompleted = 0;
+
+  //get surveys from current org
+  var results = Survey.find({organization: currentOrg}, function(err, results) {  
+    if (err) return res.status(400).send(err);
+    var callbackFinish = function() {
+      if (surveysCompleted == results.length){
+        console.log("returning");
+        return res.json(allResponses);
+      }
+    };
+    for (var i = results.length - 1; i >= 0; i--) {
+
+      Response.find({survey: results[i]._id}, function(err, responses) {
+        if (err) return res.status(400).send(err);
+        for (var j = responses.length - 1; j >= 0; j--) {
+          allResponses.push(responses[j]);
+
+        }
+        surveysCompleted++;
+        callbackFinish();
+      });  
+    }
+
+  });
+});
+
```

Comment:
That route should fail if you are an employee, it is only meant to be called by the organization.
Should I delete passport.isOrganization anyways?